testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/microprofile/restclient/RestClientProxyTest.java

```diff
@@ -63,6 +64,7 @@
       war.addClass(PortProviderUtil.class);
       war.addClass(Category.class);
       war.addClass(MicroProfileDependent.class);
+      war.addClass(NotForBootableJar.class);
```

Comment:
Why is this annotation added here?

Reply:
@ronsigal added this. I don't have it in my branch. I think it was because he noticed failures in different profiles execution (e.g.: MicroProfile). I am pushing a version that does not have the annotation, so we can see what the CI checks will report.

Reply:
@marekkopecky this test is passing now, can we resolve this conversation?